test/knativeserving.go

```diff
@@ -34,11 +34,15 @@ func CreateKnativeServing(ctx *Context, name, namespace string) (*servingoperato
 		return nil, err
 	}
 	ctx.AddToCleanup(func() error {
-		return ctx.Clients.ServingOperator.KnativeServings(namespace).Delete(serving.Name, &metav1.DeleteOptions{})
+		return DeleteKnativeServing(ctx, name, namespace)
```

Comment:
In the other PR I'm removing the one-line helpers. So perhaps we could remove it here as well and call the .Delete operation directly.